Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,55 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+
+        StartOver:
+            InteractionManager.PerformInteractions();
+
+            Console.WriteLine("Whould you like to start again?");
+            var answer = Console.ReadLine();
+            if (answer.ToLower() == "y") goto StartOver;
+        }
+    }
+
+
+
+    public static class InteractionManager
+    {
+        static string _name;
+        static int _age;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field '_age' and declare it as a local variable in the relevant methods.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4896164)